[2023-12-10 19:36:28,704] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-12-10 19:36:28,724] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-12-10 19:36:28,725] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:36:28,726] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:36:28,727] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:36:28,751] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-01-01 00:00:00+00:00
[2023-12-10 19:36:28,759] {standard_task_runner.py:52} INFO - Started process 1742 to run task
[2023-12-10 19:36:28,765] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-01-01T00:00:00+00:00', '--job-id', '460', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpqra1xdue', '--error-file', '/tmp/tmpechcr7qf']
[2023-12-10 19:36:28,768] {standard_task_runner.py:80} INFO - Job 460: Subtask segunda_tarea
[2023-12-10 19:36:28,920] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-01-01T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:36:29,104] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:00:00+00:00
[2023-12-10 19:36:29,108] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:36:29,109] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:36:29,126] {subprocess.py:85} INFO - Output:
[2023-12-10 19:36:29,130] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:36:29,131] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:36:29,217] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230101T000000, start_date=20231210T193628, end_date=20231210T193629
[2023-12-10 19:36:29,261] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:36:29,355] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
